tests/conftest.py

```diff
@@ -186,3 +188,29 @@ def telescope_model_sst(db, db_config, io_handler):
         label="test-telescope-model-sst",
     )
     return telescope_model_SST
+
+
+@pytest.fixture
+def telescope_test_file(db, args_dict, io_handler):
+    test_file_name = "telescope_positions-North-TestLayout.ecsv"
```

Comment:
Want to note that all tests seems to use `North` layouts, which is the one without SSTs. Not sure if this might cause an issue, just think about it.

Reply:
Good point. This would interfere with the entire layout array. I will open an issue on that to check how can we make sure to test both sites.